### DISCUSSION TITLE
Neopool IOBroker Integration

### DISCUSSION THREAD

**Discussion by Vippis2000** [arendst/Tasmota#21211] [Support request] [answered]
Hello,

I have compiled Tasmota with NeoPool Driver for my M5 ATOM Lite with 485Tail and got it working. In IOBroker I use sonoff adapter as MQTT broker.

In Objects I don't see and NP commands except NPFiltrationMode:

?
![Unbenannt](https://github.com/arendst/Tasmota/assets/128602940/215c44ba-32d0-4b47-a606-5dec94da99fb)

How can I change the system to get all the builtin "NPxyz" commands?

**Comment by Vippis2000** [comment]
@curzon01 @alexdelprete

**Reply by Vippis2000** [reply, marked as answer]
I solved it by changing the IOBroker mqtt adapter from sonoff to mqtt. sonoff ist not a general mqtt broker! 😃 

There you can add custom commands in the cmnd folder.

Example: 
NPFiltration with value "0" will shutdown filtration pump. "1" will start it (manual mode)

![Screenshot 2024-04-18 133856](https://github.com/arendst/Tasmota/assets/128602940/1f72a637-eee7-444b-a090-0f1ec3ffc5b9)

**Comment by alexdelprete** [comment]
Glad you solved it. Yes, a real mqtt broker is needed. :)

**Reply by Vippis2000** [reply]
Just for reminding others who may have the problem: Adding custom commands in MQTT instance of IOBroker is only possible in Expert mode!
